### ISSUE TITLE
Argument 3 passed to FOS\UserBundle\Doctrine\UserManager in sonata-project/user-bundle

### ISSUE THREAD

**Issue by Rachidelmahmoudi** [sonata-project/SonataUserBundle#1336]
**Hi,
After installing sonata-project/user-bundle I get this issue.**
Argument 3 passed to FOS\UserBundle\Doctrine\UserManager::__construct() must be an instance of Doctrine\Common\Persiste     nce\ObjectManager, instance of ContainerATvBIog\EntityManager_9a5be93 given.
This is my composer :
```json
{
    "type": "project",
    "license": "proprietary",
    "require": {
        "php": ">=7.1.3",
        "ext-ctype": "*",
        "ext-iconv": "*",
        "composer/package-versions-deprecated": "1.11.99.1",
        "doctrine/annotations": "^1.0",
        "doctrine/common": "3.1.*",
        "doctrine/doctrine-bundle": "^2.2",
        "doctrine/doctrine-migrations-bundle": "^3.0",
        "doctrine/orm": "^2.8",
        "phpdocumentor/reflection-docblock": "^5.2",
        "sensio/framework-extra-bundle": "^5.1",
        "sonata-project/admin-bundle": "3.88.0",
        "sonata-project/doctrine-orm-admin-bundle": "^3.30",
        "sonata-project/google-authenticator": "^2.3",
        "sonata-project/user-bundle": "^4.11",
        "symfony/asset": "4.4.*",
        "symfony/console": "4.4.*",
        "symfony/dotenv": "4.4.*",
        "symfony/expression-language": "4.4.*",
        "symfony/flex": "^1.3.1",
        "symfony/form": "4.4.*",
        "symfony/framework-bundle": "4.4.*",
        "symfony/http-client": "4.4.*",
        "symfony/intl": "4.4.*",
        "symfony/mailer": "4.4.*",
        "symfony/monolog-bundle": "^3.1",
        "symfony/process": "4.4.*",
        "symfony/property-access": "4.4.*",
        "symfony/property-info": "4.4.*",
        "symfony/proxy-manager-bridge": "4.4.*",
        "symfony/security-bundle": "4.4.*",
        "symfony/serializer": "4.4.*",
        "symfony/translation": "4.4.*",
        "symfony/twig-bundle": "^4.4",
        "symfony/validator": "4.4.*",
        "symfony/web-link": "4.4.*",
        "symfony/yaml": "4.4.*",
        "twig/extra-bundle": "^2.12|^3.0",
        "twig/twig": "2.13.*"
    },
    "require-dev": {
        "symfony/browser-kit": "^4.4",
        "symfony/css-selector": "^4.4",
        "symfony/debug-bundle": "^4.4",
        "symfony/maker-bundle": "^1.0",
        "symfony/phpunit-bridge": "^5.2",
        "symfony/stopwatch": "^4.4",
        "symfony/var-dumper": "^4.4",
        "symfony/web-profiler-bundle": "^4.4"
    },
    "config": {
        "preferred-install": {
            "*": "dist"
        },
        "sort-packages": true
    },
    "autoload": {
        "psr-4": {
            "App\\": "src/"
        }
    },
    "autoload-dev": {
        "psr-4": {
            "App\\Tests\\": "tests/"
        }
    },
    "replace": {
        "paragonie/random_compat": "2.*",
        "symfony/polyfill-ctype": "*",
        "symfony/polyfill-iconv": "*",
        "symfony/polyfill-php71": "*",
        "symfony/polyfill-php70": "*",
        "symfony/polyfill-php56": "*"
    },
    "scripts": {
        "auto-scripts": {
            "cache:clear": "symfony-cmd",
            "assets:install %PUBLIC_DIR%": "symfony-cmd"
        },
        "post-install-cmd": [
            "@auto-scripts"
        ],
        "post-update-cmd": [
            "@auto-scripts"
        ]
    },
    "conflict": {
        "symfony/symfony": "*"
    },
    "extra": {
        "symfony": {
            "allow-contrib": false,
            "require": "4.4.*"
        }
    }
}
```

**Comment by phansys** [comment]
@Rachidelmahmoudi, I think this issue is being addressed in #1332, but in order to confirm, please fill your bug report using the [issue template](https://github.com/sonata-project/SonataUserBundle/blob/4.x/.github/ISSUE_TEMPLATE/Bug.md), including the output of this command: `composer show --latest 'sonata-project/*'`.

**Comment by Rachidelmahmoudi** [comment]
Thank you for your response,
**`composer show --latest 'sonata-project/*'`**
```
sonata-project/admin-bundle              3.90.0 3.90.0 The missing Symfony Admin Generator
sonata-project/block-bundle              4.5.0  4.5.0  Symfony SonataBlockBundle
sonata-project/cache                     2.1.1  2.1.1  Cache library
sonata-project/datagrid-bundle           3.3.0  3.3.0  Symfony SonataDatagridBundle
sonata-project/doctrine-extensions       1.11.0 1.11.0 Doctrine2 behavioral extensions
sonata-project/doctrine-orm-admin-bundle 3.30.0 3.30.0 Integrate Doctrine ORM into the SonataAdminBundle
sonata-project/exporter                  2.5.1  2.5.1  Lightweight Exporter library
sonata-project/form-extensions           1.9.0  1.9.0  Symfony form extensions
sonata-project/twig-extensions           1.5.1  1.5.1  Sonata twig extensions
sonata-project/user-bundle               4.11.0 4.11.0 Symfony SonataUserBundle
```
**`php -v`**
```
PHP 7.3.9 (cli) (built: Feb 17 2020 12:46:14) ( NTS )
Copyright (c) 1997-2018 The PHP Group
Zend Engine v3.3.9, Copyright (c) 1998-2018 Zend Technologies
```
